packages/amplify-category-storage/src/index.ts

```diff
@@ -75,9 +79,32 @@ export async function add(context: any, providerName: any, service: any) {
   return providerController.addResource(context, AmplifyCategories.STORAGE, service, options);
 }
 
-export async function console(context: any) {
-  printer.info(`to be implemented: ${AmplifyCategories.STORAGE} console`);
-}
+/**
+ * Open AWS Management Console for resources of storage category.
+ */
+export const console = async (context: $TSContext): Promise<void> => {
+  const { amplify } = context;
+  const amplifyMeta = stateManager.getMeta();
+  if (!amplifyMeta.storage || Object.keys(amplifyMeta.storage).length === 0) {
+    return printer.error('Storage has NOT been added to this project.');
+  }
+
+  const nameOverrides = {
+    S3: 'S3 bucket - Content (Images, audio, video, etc.)',
+    DynamoDB: 'DynamoDB table - NoSQL Database',
+  };
+
+  const servicesMetadata = ((await import('./provider-utils/supported-services')) as $TSAny).supportedServices;
+
+  const serviceSelection = await amplify.serviceSelectionPrompt(context, categoryName, servicesMetadata, undefined, nameOverrides);
+
+  const providerController = await import(`${__dirname}/provider-utils/${serviceSelection.providerName}/index`);
```

Comment:
For these two dynamic imports, please leverage `path.join` instead of constructing the paths manually

For example:

```ts
  const providerController = await import(path.join(__dirname, 'provider-utils', serviceSelection.providerName, 'index'));
```